### DISCUSSION TITLE
Check hourly traffic amount from days ago?

### DISCUSSION THREAD

**Discussion by Cotopaxi42** [RaspAP/raspap-webgui#782] [Q&A]
Hi there,
I need to check the traffic amount hour by hour (same as shown on the dashboard) but from 5 days ago. I tried to check in Data usage but it doesn't show, it's juste daily traffic amount.
Would it be possible to see the hourly traffic amount from days ago? Where could I find it?
Thanks for your answer and thanks for this great project! :)

**Comment by billz** [comment]
Traffic statistics are provided by [vnstat](https://humdi.net/vnstat/). The v2.0 beta release added support for custom date range queries, e.g.:

```
vnstat -i wlan0 --days --begin 2021-01-22 --end 2021-01-24
```

However, the latest stable release version installed with `apt-get` is 1.18.

```
apt-cache madison vnstat
    vnstat |     1.18-2 | http://raspbian.raspberrypi.org/raspbian buster/main armhf Packages
```

Your best option if you want date ranges is to manually upgrade this package.

**Comment by Cotopaxi42** [comment]
Thanks for your answer, it seems a bit too complicated for me, too bad. :p

**Reply by billz** [reply]
¯\\\_(ツ)\_/¯

**Comment by Cotopaxi42** [comment]
No worries, your soft it great anyway. ;)
